tests/ci/file.go

```diff
@@ -0,0 +1,269 @@
+package ci
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/user"
+	"path"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	CON "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
+	EXE "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/exec"
+	f "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/files"
+	h "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/helper"
+)
+
+// The cfg will be used to define the testing environment
+var cfg = RHCS
+
+func HomeDir() string {
```

Comment:
We don't need to record file on local. Terraform will record files on local. Remove all records part. And we don't need to do e2e for now. Just SDK is required.